fastlane/Fastfile

```diff
@@ -428,6 +429,70 @@ platform :android do
     )
   end
 
+  desc 'Builds a prototype build and uploads it to S3'
+  lane :build_and_upload_prototype_build
+    UI.user_error!("'BUILDKITE_ARTIFACTS_S3_BUCKET' must be defined as an environment variable.") unless ENV['BUILDKITE_ARTIFACTS_S3_BUCKET']
+
+    gradle(
+        task: "assembleRelease",
+        properties: {
+        "skipSentryProguardMappingUpload" => true
+      }
+    )
+
+    prototype_build_comments = lane_context[SharedValues::GRADLE_ALL_APK_OUTPUT_PATHS].map { |apk_path|
+      upload_path = upload_to_s3(
+        bucket: 'a8c-apps-public-artifacts',
+        key: "pocketcasts-mobile-prototype-build-#{generate_prototype_build_number}.apk",
+        file: apk_path,
+        skip_if_exists: true
+      )
+
+      install_url = "#{PROTOTYPE_BUILD_DOMAIN}/#{upload_path}"
+
+      prototype_build_details_comment(
+        app_display_name: get_app_display_name(apk_path: apk_path),
+        download_url: install_url,
+        fold: true
+      )
+    }
+
+    unless ENV['BUILDKITE_PULL_REQUEST'].nil?
+        comment_on_pr(
+            project: 'automattic/pocket-casts-android',
+            pr_number: Integer(ENV['BUILDKITE_PULL_REQUEST']),
+            reuse_identifier: "app-prototype-build-link",
+            body: prototype_build_comments.join('\n')
+        )
+  end
+
+  def get_app_display_name(apk_path:)
+    if apk_path.include?('app')
+      "📱 Mobile"
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/pocket-casts-android/blob/14fbc28fd1fb46d68ff9627a26aa8717dc4f1c77/fastlane/Fastfile#L471"></span>Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.</td>
    </tr>
  </tbody>
</table>